### PR TITLE
Clean up wording in "Add, then delete" section

### PR DESCRIPTION
**Pull request by dannyroberts**
## Overview

This section of the documentation was oddly worded and difficult to follow. This just cleans that up a little.

### CODE REVIEW

File: src/cluster/sharding.rst [hunk 1]
```diff
@@ -198,19 +198,20 @@ Moving Shards
...
 #. Delete the shard on the old node.
-#. Lose all writes made between 1 and 2.
+#. Oh, no!: You have lost all writes made between 1 and 2.
 
-As the reality "I added a new replica of the shard X on node Y and then I waited
-for them to sync, before I removed the replica of shard X from node Z." is a bit
-tedious, people and this documentation tend to use the illusion of moving.
+To avoid this mistake, you always want to make sure that both shards have been live for some time
+and that the shard on your new node is fully caught up before removing a shard on an old node.
+Since "moving" a more conceptually—if not technically—accurate of what you want to do,
```
Comment (flimzy) [inline src/cluster/sharding.rst]: Should be `Since "moving" is a more conceptually...` ?

File: src/cluster/sharding.rst [hunk 2]
```diff
@@ -296,5 +297,5 @@ any real gain. You might even get lower performance. As an example of this, we
 can take the author's (15 year) old lab server. It gets noticeably slower with
 more than one shard and high load, as the hard drive must seek more.
 
-How many shards you should have depends, as always, on your use case and your
+How many shards you should X depends, as always, on your use case and your
```
Comment (flimzy) [inline src/cluster/sharding.rst]: Not sure what this change was meant to represent... "have" makes more sense to me?

**Comment by flimzy** [comment]
Content looks good now, thanks for the follow-up.  All that remains now is to get the checks to pass. There are some long lines causing the failures. See [here](https://travis-ci.org/apache/couchdb-documentation/jobs/295769483#L512).

**Comment by wohali** [comment]
Since we're pushing forward with a 2.1.1 RC2 today, I'm merging this so I can move the tag and rebuild.

**Pull request merged**